docs/contributing/Build-XKit.md

```diff
@@ -1,63 +1,91 @@
+While you can use the XKit Editor to make and test minor changes to an individual extension with minimal setup, as described in [Fixing a Bug](./Fixing-a-Bug.md), loading a development build of XKit allows you to make and test changes to XKit itself and can enable faster iteration of extension modifications.
+
 ## Prerequisites:
 
 * Download [Node.js](https://nodejs.org/download/) for your platform.
```

Comment:
This could also specify that we expect the build tools to work with the latest LTS (current thing for the startup is poking around Python projects so I'm a bit sensitive to documenting compatible versions right now lol)

Reply:
Makes sense to me. Any suggestions about phasing?

Reply:
As any experienced player knows, there are multiple valid places to phase. The right one depends on your DPS (documentation per sentence) :3